website/docs/configuration/snippet-choice.md

```diff
@@ -0,0 +1,187 @@
+The snippets are short pieces of code coming from difference public sources such as GitHub, GitLab, Stackoverflow, ...
```

Comment:
Note: I'll skip reviewing the docs for now an focus on the code.

Reply:
I believe this section should also have a header, like `# Introduction`.

Reply:
"The snippets" -> "Snippets"

Reply:
I'd stop the sentence after "code" and start a new one like "They might come from ..." (because not only pieces of code coming from public sources are called snippets).

Reply:
"..." -> "etc."

Reply:
I had to change the section titles because of https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md025.md.